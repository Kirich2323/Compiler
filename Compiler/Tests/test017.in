'simple string'
''
'string with'#10' espace symbols'''