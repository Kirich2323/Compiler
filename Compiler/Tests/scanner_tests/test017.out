1    1   'simple string' simple string   String
2    1   ''                              String
3    1   'string with'#10' espace symbols''' string with
 espace symbols' String
